src/main/java/Entities/RecipeBook.java

```diff
@@ -1,7 +1,9 @@
 package main.java.Entities;

```

Comment:
for the RecipeBook file, there is an updated file in the pull request #81 including the filters and sort functions